raml2markdown/src/identity-api/identity-api.raml

```diff
@@ -35,10 +35,26 @@ types:
       type:
         description: The type of the link
         type: string
-/identity:
+/:
   description: |
-    Manage identities. Create, read, update, delete and list identities from
-    your world.
+    Create a new or list identities. If optional `type` GET-parameter is given,
```

Comment:
How does it look like if we move some of this description purely under the `GET` and leave this e.g. as `Manage identities`?

Reply:
Well, since there is now multiple "main blocks" for endpoints, the `/` only handles POST and GET, which is not really "managing" the identities, as in turn `/<id>` has PUT, DELETE, GET. It looks more like "managing" identities than the root.

Then again, I think it totally depends on how it looks on the website, as I've only used the raml2html default styles to write the descriptions and checking how that looks like.